### PR TITLE
Added ScreenSaver for sugarizer

### PR DESCRIPTION
**Pull request by crusher95**

### CODE REVIEW

**Comment by davelab6** [comment]
@crusher95 please rebase your fork and fixup the 2nd commit into a single commit :)

https://www.google.com/search?q=git+rebase+interactive

**Comment by davelab6** [comment]
What is the license for `js/screensaver.jpg`? I think this is an image subject to restrictive copyrights from threadless.com

**Comment by davelab6** [comment]
Why is css/styles.css mod from 100644 → 100755?

The only changes to css/styles.css relevant to this PR seem to be

```
#screen{
	z-index: 100000;
}
```

The code-style can be improved (adding a space before `{`) 

@llaske do you have a code style linter?

File: css/styles.css [hunk 1]
```diff
@@ -1112,4 +1107,4 @@ input {
 	.journal-filter-time {
 		visibility: visible;
 	}
-}
+}
```
Comment (davelab6) [inline css/styles.css]: It is good practice to have a final blank newline at the end of every text line to prevent concatenated output merging the last and first lines of files.

File: index.html [hunk 2]
```diff
@@ -35,9 +35,10 @@
 	<script src="js/searchfield.js" type="text/javascript"></script>	
 	<script src="js/selectbox.js" type="text/javascript"></script>	
 	<script src="js/dialog.js" type="text/javascript"></script>	
+	<script type="text/javascript" src="js/screenSaver.js"></script>
   </head>
   
-  <body>
+  <body onload="screenSaver()" id="body">
```
Comment (davelab6) [inline index.html]: `id="body"` is not needed because the element is `<body>`

File: css/styles.css [hunk 3]
```diff
@@ -126,7 +126,6 @@ input {
 .search-field-input {
 	opacity: 1;
 	margin-left: 30px;
-	margin-top: 5px;
```
Comment (davelab6) [inline css/styles.css]: why is this changed?

File: css/styles.css [hunk 4]
```diff
@@ -152,12 +151,6 @@ input {
 .web-activity-icon {
 	background-repeat: no-repeat;	
 	background-position: center;
-	padding: 2px;
-}
-
-.web-activity-icon:hover {
-	background-color: #d5d5d5;
-	border-radius: 5px;
```
Comment (davelab6) [inline css/styles.css]: why is this changed?

File: css/styles.css [hunk 5]
```diff
@@ -152,12 +151,6 @@ input {
 .web-activity-icon {
 	background-repeat: no-repeat;	
 	background-position: center;
-	padding: 2px;
```
Comment (davelab6) [inline css/styles.css]: why is this changed?

File: css/styles.css [hunk 6]
```diff
@@ -152,12 +151,6 @@ input {
...
-}
-
-.web-activity-icon:hover {
-	background-color: #d5d5d5;
```
Comment (davelab6) [inline css/styles.css]: why is this changed?

File: css/styles.css (hunk 1 above)
Comment (davelab6) [inline css/styles.css]: Please add a blank new line to the end of the file

File: css/styles.css [hunk 7]
```diff
@@ -942,7 +935,9 @@ input {
 	margin-left: auto;
 	margin-right: auto;
 }
-
```
Comment (davelab6) [inline css/styles.css]: The code style of the file is to have a blank line between each CSS rule. Please add this back.

File: js/screenSaver.js [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+}
+function deactivate(){
+	clearTimeout(Timer);
+	var parent=document.getElementsByTagName("body");
+	var child=document.getElementById("screen");
+	child.style.display="none";
+	Timer=setTimeout(resume,timelimit);
+}
+function resume(){
+	var image=document.getElementById("screen");
```
Comment (davelab6) [inline js/screenSaver.js]: you have

```
var child=document.getElementById("screen");
```

and then
```
var child=document.getElementById("screen");
```

For me this is confusing, since the file thing is named differently. I suggest using the same name for the same thing.

**Comment by davelab6** [comment]
965a5f8 adds a new gif, but what is the license for this?  Where is it from?

**Comment by llaske** [comment]
Don't work here. image is null line 12 of the screensaver.js script both on Chrome and Firefox.
Plus I'm not at all a big fan of the idea of a screen saver. It has no sense on devices where screen saving is already handle (tablet, smartphone and PC) and I don't see the interest of a screen saver in a web  browser.

**Comment by crusher95** [comment]
Hi,

Yes point accepted (y) . Is there any other issue that you could assign me
or suggest me for sugarizer?

Thanking You,
Utkarsh Dhawan
(Developer/Student)

Note:Please Consider The environment before printing.

**Comment by davelab6** [comment]
I think this is the reason: It is fun! :)

**Comment by llaske** [comment]
Hi Utkarsh, I don't have any issue to assign, except if you find one in testing.
BTW, if you've got some time, may be you could try to port some existing activities. Why not FractionBounce ? http://activities.sugarlabs.org/en/sugar/addon/4488

**Comment by crusher95** [comment]
Hi,

An interesting activity indeed. I am on it (y)

Thanking You,
Utkarsh Dhawan
(Developer/Student)

Note:Please Consider The environment before printing.
